licenses/BSL.txt

```diff
@@ -7,7 +7,7 @@ Parameters
 Licensor:                   Democratized Data (D2) Foundation
 
 
-Licensed Work:              DefraDB v0.11.0
+Licensed Work:              DefraDB v0.13.0
```

Comment:
:)

Reply:
I know... That and the changelog file copy that got forgotten 🤦‍♂️